src/views/admin-results-listing.phtml

```diff
@@ -46,7 +63,7 @@ require_once('partials/head.phtml');
                     <?php /** @var $result UserQuizResult */ ?>
                     <th scope="row"><?php echo  $result->getQuizInstance()->getId(); ?></th>
                     <td><?php echo $result->getUser()->getName() ?> </td>
-                    <td><?php echo $result->getQuizInstance()->getQuizTemplateId() ?></td>
+                    <td><?php echo $result->getQuizInstance()->getName() ?></td>
                     <td>We don't believe that numbers represent the value of people.</td>
```

Comment:
you should remove this eventually :))

Reply:
I've changed it in "Add score" PR :))) @Raresmldvn